gamla/functional_generic.py

```diff
@@ -324,7 +324,16 @@ def ternary_inner(*args, **kwargs):
     return ternary_inner
 
 
-def when(condition, f_true):
+def when(condition: Callable, f_true: Callable) -> Callable:
+    """Computes `f_true` with the given arguments if `condition` with the
+    same arguments return true, else returns the given arguments.
```

Comment:
```suggestion
    """Returns `f_true(args)` if `condition(args)` returns true, else returns args.
```
(just a suggestion :))

Reply:
I'll take it